cli/js/lib.deno.ns.d.ts

```diff
@@ -390,8 +405,7 @@ declare namespace Deno {
   export function execPath(): string;
 
   /**
-   * **UNSTABLE**: Currently under evaluation to decide if explicit permission is
-   * required to get the value of the current working directory.
+   * **UNSTABLE**: Need allow-read check, then is stable.
```

Comment:
To which path is read permission needed for `Deno.cwd()`? How might you request it?

Reply:
`Deno.permissions.request({ name: "read", path: "." })`, got it 👍